pkg/authorization/authorizer/scope/converter.go

```diff
@@ -178,37 +179,37 @@ func (userEvaluator) Describe(scope string) (string, string, error) {
 	}
 }
 
-func (userEvaluator) ResolveRules(scope, namespace string, clusterPolicyGetter authorizationlister.ClusterPolicyLister) ([]authorizationapi.PolicyRule, error) {
+func (userEvaluator) ResolveRules(scope, namespace string, _ rbaclisters.ClusterRoleLister) ([]rbac.PolicyRule, error) {
 	switch scope {
 	case UserInfo:
-		return []authorizationapi.PolicyRule{
-			{Verbs: sets.NewString("get"), APIGroups: []string{userapi.GroupName, userapi.LegacyGroupName}, Resources: sets.NewString("users"), ResourceNames: sets.NewString("~")},
+		return []rbac.PolicyRule{
```

Comment:
I have an ultrawide monitor, that is not the problem.
There has been a ton of research on line lengths over time:
http://www.humanfactors.com/newsletters/optimal_line_length.asp
Beyond the speed numbers I personally feel it much harder to comprehend code that has long lines, especially without syntax in between, so much so I sometimes have to break the code in the editor myself when such long lines are there.
Given I had to go through the tedious task of making sure I was not accidenteally changing rules when I was translating those forms I tried to use the form that made it very clear what arguments were being passed there, so I would not accidentally miss something at character number 240 when the source linewrapped and was 240 chars to the left instead causing my eyes to have to keep jumping 10" left and right in the process to double check.
Sorry if I seem to make a big deal of this, but it is a big deal to me to be able to *read* code after or while I write it :-)